fix: Remove encoding character issue beside "PowerShell" (#1084)
MicrosoftDocs/windows-powershell-docs: docset/winserver2012-ps/grouppolicy/GroupPolicy.md

# GroupPolicy Module
## Description

> This topic contains the brief descriptions of the Windows PowerShell cmdlets that are for use in administering Group Policy in Windows Server and Windows client with Remote Server Administration Tools (RSAT) installed. (RSAT includes the GPMC and the Group Policy cmdlets.)

Reviewer: ```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets that used to administer Group Policy in Windows Server and Windows client with Remote Server Administration Tools (RSAT) installed. (RSAT includes the GPMC and the Group Policy cmdlets).
```
